CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [9.1.x] - 2025-05-??
 
+### Metroid Dread
+
+- Changed: The maximum number of Speed Booster Upgrade items that will have an effect is now 4 instead of 5 (any further upgrades have no effect). This is to prevent a quirk with inconsistent Speed Booster activation with very short charge times.
+- Fixed: Speed Booster Upgrade items will no longer be hinted or included in the credits.
```

Comment:
They can still be hinted at. But its a *much* lower chance now.

Reply:
The chance is effectively 0

Reply:
Was mostly thinking to prevent people reporting that as a bug when they read the changelog and try to craft a preset to specifically trigger them

But i guess the chance of that happening is also effectively 0 lol